src/proto/Cosmos.proto

```diff
@@ -272,16 +272,16 @@ message Message {
 
     // VoteOption enumerates the valid vote options for a given governance proposal.
     enum VoteOption {
-        // VOTE_OPTION_UNSPECIFIED defines a no-op vote option.
-        VOTE_OPTION_UNSPECIFIED = 0;
-        // VOTE_OPTION_YES defines a yes vote option.
-        VOTE_OPTION_YES = 1;
-        // VOTE_OPTION_ABSTAIN defines an abstain vote option.
-        VOTE_OPTION_ABSTAIN = 2;
-        // VOTE_OPTION_NO defines a no vote option.
-        VOTE_OPTION_NO = 3;
-        // VOTE_OPTION_NO_WITH_VETO defines a no with veto vote option.
-        VOTE_OPTION_NO_WITH_VETO = 4;
+        //_UNSPECIFIED defines a no-op vote option.
+        _UNSPECIFIED = 0;
```

Comment:
Unspecified was already use, using _ as prefix.

Reply:
maybe we should remove this? what's the point to vote without yes/no

Reply:
You can override a vote, if you override with unspecified this means you let your validator vote for u.